docs-src/strongly-typed-errors/bad-cancel-timer-attributes.md

```diff
@@ -0,0 +1,14 @@
+---
+id: bad-cancel-timer-attributes
+title: Bad Cancel Timer Attributes
+description: Explanation for Bad Cancel Timer Attributes error message, and how to fix it.
+sidebar_label: Bad Cancel Timer Attributes
+tags:
+  - errors
+  - strongly-typed
+---
+
+This error indicates that the Workflow Task failed while attempting to cancel a Timer.
+
+Check your Timer attributes for a missing Timer Id value.
+Add a valid Timer Id and retry the Task.
```

Comment:
What does it mean to "retry the Task" in this context?

Are we talking about fixing code, redeploying it, and waiting for the Worker to pick it up?

Reply:
Asking about this. This is one of the things I wanted more detail on.